content/docs/details/scaffolder/index.md

```diff
@@ -84,247 +91,261 @@ Each parameter can be one of a few types: `string`, `number`, `array` or `object
+parameters:
+  properties:
+    name:
+      type: string
+steps:
+  - id: log-message
+    name: Log Message
+    action: debug:log
+    input:
+      message: 'Hello, ${{ parameters.name }}!'
 ```
 
 If you need to reference elements of an array parameter you can refer to them using the following syntax:
 
 ```yaml
-  steps:
-    - id: log-message
-      name: Log Message
-      action: debug:log
-      input:
-        message: 'Hello, ${{ parameters.names[0] }}!'
+steps:
+  - id: log-message
+    name: Log Message
+    action: debug:log
+    input:
+      message: 'Hello, ${{ parameters.names[0] }}!'
 ```
 
 An `object` parameter values can be reference in the way you might expect.
 
 ```yaml
-  steps:
-    - id: log-message
-      name: Log Message
-      action: debug:log
-      input:
-        message: 'Hello, ${{ parameters.person.name }}!'
+steps:
+  - id: log-message
+    name: Log Message
+    action: debug:log
+    input:
+      message: 'Hello, ${{ parameters.person.name }}!'
 ```
 
 ### Common Options
 
 If you would like to default the value of a field you can use the `default` option:
 
 ```yaml
-  parameters:
-    properties:
-      name:
-        type: string
-        default: "world!"
+parameters:
+  properties:
+    name:
+      type: string
+      default: 'world!'
 ```
 
 If you would like to prompt the users for a fixed list of options, you may use the `enum` option.
 
 ```yaml
-  parameters:
-    properties:
-      size:
-        type: number
-        enum: [50, 100, 200]
+parameters:
+  properties:
+    size:
+      type: number
+      enum: [50, 100, 200]
 ```
 
 You can display a more human description to a field value by using `title` and `description`
 
 ```yaml
-  parameters:
-    properties:
-      name:
-        type: string
-        title: "Name"
-        description: "Name to say hello to"
+parameters:
+  properties:
+    name:
+      type: string
+      title: 'Name'
+      description: 'Name to say hello to'
 ```
 
 ### Form Steps
+
 It might be jarring for your user to enter a lot of parameters one after another on the same page, especially if some of the properties require validation. As such Backstage have provided form steps.
 
 You can make use of form steps using the following example.
 
 ```yaml
-  parameters:
-    - title: "Fill in the Name"
-      properties:
-        name:
-          type: string
-    - title: "Fill in the Age"
-      properties:
-        age:
-          type: number
+parameters:
+  - title: 'Fill in the Name'
+    properties:
+      name:
+        type: string
+  - title: 'Fill in the Age'
+    properties:
+      age:
+        type: number
 ```
 
+### Previewing parameters
+
+Tools section provides a preview page for templates, where you can see a live preview of the template form. This is done in order to provide an easy way to preview scaffolder template form UIs without running your own local instance of the plugin or committing changes to the template.
```

Comment:
Where is the tools section?

Reply:
There is an image below where it is visible it is under Administration. Do you think it is worth mentioning it in the text as well?

Reply:
I think if you don't then the reader won't be able to find it.